README.adoc

```diff
@@ -87,11 +84,15 @@ If you already have Docker installed, make sure to have it running.
 https://azure.microsoft.com/en-us/pricing/purchase-options/pay-as-you-go/[Microsoft Azure Purchase Options^] 
 to create an account with your email and start a Pay-As-You-Go subscription. 
 
+* *IBM WebSphere Liberty and Open Liberty on AKS:* You will need to use the IBM WebSphere Liberty and Open Liberty on AKS on Azure marketplace. See the official
```

Comment:
Is "WebSphere Liberty/Open Liberty on Kubernetes" the official name instead of "IBM WebSphere Liberty and Open Liberty on AKS"?